async-race/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "svg.preview.background": "editor"
+}
```

Comment:
![image](https://user-images.githubusercontent.com/33457473/218389622-168da750-3f16-44ec-afb9-edd6e2088c54.png)
Maybe we have clean dist?